modules/Tasks/Task.php

```diff
@@ -264,12 +249,12 @@ protected function formatStartAndDueDates(&$task_fields, $dbtime, $override_date
         $taskClass = 'futureTask';
 		if ($dd < $today){
             if($task_fields['STATUS']=='Completed' || $task_fields['STATUS']=='Deferred')
-			{ 
+			{
 				$taskClass = '';
-			} 
```

Comment:
What's with this change?

Reply:
Looks like it's just whitespace, I probably hit the auto-format button while I was in this file.